calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,169 @@
+| `calibration_metrics`         | `std_msgs::msg::Float32MultiArray`     | Calibration metrics.                            |
+
+### Services
+
+| Name                       | Type                                                  | Description                                                                              |
+| -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration`    | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `extract_background_model` | `std_srvs::srv::Empty`                                | Starts to extract the background model from radar and lidar data.                        |
+| `add_lidar_radar_pair`     | `std_srvs::srv::Empty`                                | Adds lidar-radar pairs for calibration.                                                  |
+| `delete_lidar_radar_pair`  | `std_srvs::srv::Empty`                                | Deletes the previous lidar-radar pair.                                                   |
+| `send_calibration`         | `std_srvs::srv::Empty`                                | Sends the calibration result to the sensor calibration manager.                          |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                        | Type          | Default Value                                           | Description                                                                                                                                                        |
+| ------------------------------------------- | ------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `radar_parallel_frame`                      | `std::string` | `base_link`                                             | The reference frame to which the radar optimizes its transformation.                                                                                               |
+| `msg_type`                                  | `std::string` | `radar tracks`/`radar scan`                             | The type of the input radar objects. (Not available yet, currently only support radar tracks.)                                                                     |
+| `transformation_type`                       | `std::string` | `yaw_only_rotation_2d`/`svd_2d`/`svd_3d`/`roll_zero_3d` | Specifies the algorithm used to optimize the transformation between the radar frame and the radar parallel frame. (Not available yet.)                             |
```

Comment:
See how this looks on the rendered markdown.
Split the list so that they do not consume the whole width of the page, this has already been done in other parts of the documentation if you do not know how to do it

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/b38fc0c3241ca0e1502dd4555196f63df4e6d7fd